server/routes.js

```diff
@@ -548,3 +548,34 @@ exports.mountQuintypeAt = function(app, mountAt) {
     }
   });
 };
+
+/**
+ * ampRoutes uses quintype-amp library to handle amp pages
+ * currently, only story pages with default styling are supported
+ */
+exports.ampRoutes = function(app, { opts } = {}) {
+  const { Story } = require("./api-client");
+  const { ampifyStory } = require("@quintype/amp");
+  getWithConfig(
+    app,
+    "/amp/story/*",
+    async (req, res, next, { client, config }) => {
+      try {
+        const slug = req.path.replace("^/amp/story", "");
+        const ampConfig = await client.getAmpConfig();
+        const story = await Story.getStoryBySlug(client, slug);
+        const ampHtml = ampifyStory({
+          story,
+          config: { ...ampConfig, ...config },
```

Comment:
Why are you merging these two into a single config instead of passing it as two keys? This makes it tough to debug as you won't know where any config is coming from.

Reply:
Yeah, that makes sense